Palace.xcodeproj/project.pbxproj

```diff
@@ -4511,7 +4511,6 @@
 				CLANG_WARN_NON_LITERAL_NULL_CONVERSION = YES_ERROR;
 				CODE_SIGN_ENTITLEMENTS = Palace/SimplyE.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Distribution";
-				CURRENT_PROJECT_VERSION = 142;
```

Comment:
Could you check the build number here before merging the branch? It looks like the build number disappeared

Reply:
Nice catch. I resolved a merge conflict and must have lost them in the resolution.